### ISSUE TITLE
[TYPO] Chapter 14 CNNs

### ISSUE THREAD

**Issue by kasrahabib** [ageron/handson-ml2#485]
@ageron

Maybe a typo in the book. Please have a look at Chapter 14, Tensorflow Implementation Subtitle:

In the code:

``` python
outputs = tf.nn.conv2d(images, filters, strides=1, padding="same")
```

In the descriptions:

"We apply them to both images using the ```tf.nn.conv2d()``` function, which is part of TensorFlow’s low-level Deep Learning API. In this example, we use zero padding (padding="same") and a stride of 2."

Shouldn't the stride be 1 instead of 2 in the description?

**Comment by ageron** [comment]
Thanks @kasrahabib, you're absolutely right. I fixed this issue a few months ago, so the newer releases are now fixed. 👍 
I hope you're enjoying the book!
